packages/rancher-monitoring/overlay/templates/rancher-monitoring/dashboard-role.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if and .Values.global.rbac.create .Values.global.rbac.userRoles.create .Values.grafana.enabled }}
+{{- if or .Values.grafana.sidecar.dashboards.enabled .Values.grafana.defaultDashboardsEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: monitoring-dashboard-admin
+  namespace: {{ .Values.grafana.sidecar.dashboards.searchNamespace }}
+  labels: {{ include "kube-prometheus-stack.labels" . | nindent 4 }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: monitoring-dashboard-edit
+  namespace: {{ .Values.grafana.sidecar.dashboards.searchNamespace }}
+  labels: {{ include "kube-prometheus-stack.labels" . | nindent 4 }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - '*'
```

Comment:
Should this be star? That would allow them to delete and create. This applies to the other roles as well if it needs changed.

Reply:
Yes, a user with edit permissions should have the ability to delete and create as well; for example, this would give someone with `monitoring-dashboard-edit` permissions to persist their Grafana Dashboards (create) and also remove some dashboards that they had added that no longer need to be persisted (delete).

I believe we discussed this in a previous design meeting, but there's really no difference between a user with admin vs. edit permissions at the moment.